resync/settings_validators.py

```
@@ -301,12 +341,23 @@ def validate_http_pool_sizes(cls, v: int, info: ValidationInfo) -> int:
+            raise ValueError(
+                f"redis_max_connections ({v}) must be >= redis_min_connections ({min_size})"
+            )
+        return v
+
     # =========================================================================
     # MODEL-LEVEL VALIDATOR (runs after all field validators)
     # =========================================================================
 
     @model_validator(mode="after")
-    def validate_cross_field_consistency(self):
+    def validate_cross_field_consistency(self) -> "SettingsValidators":
```

Ruff (0.15.0)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)